offchain/rollups-events/tests/tls.Dockerfile

```diff
@@ -0,0 +1,65 @@
+# Copyright Cartesi Pte. Ltd.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may not
+# use this file except in compliance with the License. You may obtain a copy of
+# the License at http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations under
+# the License.
+
+FROM rust:1.67.0-bullseye AS chef
+
+ENV CARGO_REGISTRIES_CARTESI_INDEX=https://github.com/cartesi/crates-index
+RUN rustup component add rustfmt
+RUN cargo install cargo-chef
+
+FROM chef AS planner
+
+COPY . /usr/src/offchain
+WORKDIR /usr/src/offchain
+RUN cargo chef prepare --bin rollups-events --recipe-path recipe.json
+
+FROM chef AS builder
+
+RUN <<EOF
+DEBIAN_FRONTEND="noninteractive" apt-get update
+DEBIAN_FRONTEND="noninteractive" apt-get install -y --no-install-recommends openssl
+EOF
+
+COPY --from=planner /usr/src/offchain/recipe.json recipe.json
+RUN cargo chef cook --release --recipe-path recipe.json
+
+COPY . /usr/src/offchain
+WORKDIR /usr/src/offchain/rollups-events
```

Comment:
Could we reuse part of the dockerfile from `offchain/`? It takes a long time to build this image and I wish to reuse the cached version.

Reply:
I'm not sure if that is the best solution. However, the tests took 10 min to run on my computer, which is too much.

Reply:
As time is of the essence, I'll move the test code to another PR so this one can be merged and @endersonmaia can start testing on his side. Then we can keep discussing how to improve the test in the new PR.

Reply:
Created #77